src/WalkingModule/src/Module.cpp

```diff
@@ -598,13 +723,54 @@ bool WalkingModule::updateModule()
         }
 
         // evaluate 3D-LIPM reference signal
-        m_stableDCMModel->setInput(m_DCMPositionDesired.front());
+        if(m_useStepAdaptation)
+        {
+            if (!m_leftInContact.front() || !m_rightInContact.front())
+            {
+                if(!m_leftInContact.front()){
+                    if (!feetTrajectorySmoother(m_adaptedFootLeftTransform,m_leftTrajectory.front(),m_smoothedFootLeftTransform,m_adaptedFootLeftTwist,m_leftTwistTrajectory.front(),m_smoothedFootLeftTwist)) {
+                        yError()<<"[WalkingModule::updateModule] The left foot trajectory smoother can not evaluate the smoothed trajectory!";
+                    }
+                }
+
+                if(!m_rightInContact.front())
+                {
+                    if (!feetTrajectorySmoother(m_adaptedFootRightTransform,m_rightTrajectory.front(),m_smoothedFootRightTransform,m_adaptedFootRightTwist,m_rightTwistTrajectory.front(),m_smoothedFootRightTwist)) {
+                        yError()<<"[WalkingModule::updateModule] The right foot trajectory smoother can not evaluate the smoothed trajectory!";
+                    }
+                }
+            }
+            else
+            {
+                m_smoothedFootLeftTransform=m_adaptedFootLeftTransform;
+                m_smoothedFootLeftTwist=m_adaptedFootLeftTwist;
+
+                m_smoothedFootRightTransform=m_adaptedFootRightTransform;
+                m_smoothedFootRightTwist=m_adaptedFootRightTwist;
+            }
+
+            if (!dcmSmoother(m_DCMPositionAdjusted.front(),m_DCMPositionDesired.front(),m_DCMPositionSmoothed))
+            {
+                yError()<<"the DCM smoother can not evaluate the smoothed DCM!";
+            }
+            m_stableDCMModel->setInput(m_DCMPositionSmoothed);
+        }
+        else {
```

Comment:
Please be compliant with the style
```suggestion
        else 
        {
```

Reply:
solved!